src/ucp/tag/rndv.c

```diff
@@ -214,6 +240,40 @@ static void ucp_rndv_send_atp(ucp_request_t *sreq, uintptr_t remote_request)
     ucp_request_send(sreq, 0);
 }
 
+UCS_PROFILE_FUNC_VOID(ucp_rndv_complete_frag_rma_put_zcopy, (fsreq),
+                      ucp_request_t *fsreq)
+{
+    ucp_request_t *sreq = fsreq->send.rndv_put.sreq;
```

Comment:
you mean struct member name ? can you elaborate a bit more

Reply:
`fsreq` should be `req->send->proto` now right? can't access `req->send.rndv_put` then

Reply:
No. it is still `req->send.rndv_put `https://github.com/openucx/ucx/blob/master/src/ucp/core/ucp_request.h#L163

Reply:
In `ucp_rndv_send_frag_atp` you initialize `fsreq->send.proto`. When it completes it calls `ucp_rndv_complete_frag_rma_put_zcopy`, where you try to access `fsreq->send.rndv_put`

Reply:
Ah. my bad!.  it seems it worked because it has taken "send_tag" field of proto.  
Fixed by adding new sreq filed to proto as it still has room for extra field w/o changing the size.